### PR TITLE
doc: Setting up KBL serial console on the GSG

### PR DESCRIPTION
**Pull request by lirui34**
Add descriptions of setting up serial console for KBL NUCs.

Signed-off-by: ruix.li <ruix.li@intel.com>

### CODE REVIEW

**Comment by NanlinXie** [comment]
Can one of the admins verify this patch?

**Comment by acrnsi** [comment]
OK to verify

**Comment by wenlingz** [comment]
@dbkinder please help review this PR which add "set up KBL serial console on the GSG" part, thanks

File: doc/getting-started/apl-nuc.rst [hunk 1]
```diff
@@ -3,38 +3,42 @@
...
+  <https://www.amazon.com/dp/B07D5HHMSB/ref=cm_sw_r_cp_ep_dp_hXm0BbBV0CER3>`__
+  (NUC7i5DNHE), but this is not supported on APL (NUC6CAYH).
 
 
 Connecting to the serial port
 =============================
 
-You can just ignore this section if command line is not needed via the serial console.
-Or you have to prepare these cables to enable the serial port.
+You can just ignore this section if you don't need serial console.
```
Comment (dbkinder) [inline doc/getting-started/apl-nuc.rst]: Change to:
```
If you don't need a serial console you can ignore this section.  If you're 
using a KBL NUC and you need a serial console, you'll need to prepare
an RS232 cable to connect to the KBL NUC serial port header, as shown
below:
```

File: doc/getting-started/apl-nuc.rst [hunk 2]
```diff
@@ -206,20 +210,27 @@ partition. Follow these steps:
       #. ``uart=port@<port address>``: this sets the serial port address
 
       .. note::
-         
+
          ``uart=port@<port address>`` is required if you want to enable the serial console.
          You should run ``dmesg |grep ttyS0`` to get port address from the output, and then
-         set the ``uart`` parameter as ``uart=port@0x3f8``
-         
+         add ``uart`` parameter into ``efibootmgr`` command.
```
Comment (dbkinder) [inline doc/getting-started/apl-nuc.rst]:
```suggestion
         add the ``uart`` parameter into the ``efibootmgr`` command.
```

**Comment by lirui34** [comment]
@dbkinder Thank you for the comments, the doc is updated.

**Comment by dbkinder** [comment]
@gvancuts I'll merge this and we can submit a fix if you find anything else...

**Comment by gvancuts** [comment]
> @gvancuts I'll merge this and we can submit a fix if you find anything else...

This looks fine to me, thanks @dbkinder and @lirui34

**Pull request merged**